server/repository/src/db_diesel/program_requisition/program_requisition_order_type_row.rs

```diff
@@ -80,4 +80,23 @@ impl<'a> ProgramRequisitionOrderTypeRowRepository<'a> {
 
         Ok(result)
     }
+
+    pub fn find_many_by_requisition_settings_id(
```

Comment:
there was another one created at the same time, probably best to delete one of the duplicates